apps/mark-scan/backend/src/app.ts

```diff
@@ -158,6 +173,13 @@ function buildApi(
       });
       workspace.store.setSystemSettings(systemSettings);
 
+      const { precincts } = electionDefinition.election;
+      if (precincts.length === 1) {
+        workspace.store.setPrecinctSelection(
+          singlePrecinctSelectionFor(precincts[0].id)
+        );
+      }
```

Comment:
This is existing behavior in the frontend that I ported to the backend. See: Deleted test `apps/mark-scan/frontend/src/app_single_precinct_election.test.tsx`